honeybee_schema/energy/ventcool.py

```diff
@@ -103,3 +104,168 @@ class VentilationOpening(NoExtraBaseModel):
+        default=0.7,
+        ge=0.5,
+        le=1,
+        description='An optional dimensionless number between 0.5 and 1 used to '
+        'calculate the mass flow rate when the opening is closed; required to run an '
+        'AirflowNetwork simulation. This value represents the leak geometry impact '
+        'on airflow, with 0.5 generally corresponding to turbulent orifice flow and 1 '
+        'generally corresponding to laminar flow. The default of 0.7 is representative '
+        'of internal and external window leakage, according to the DesignBuilder Cracks '
+        'template.'
+    )
+
+    minimum_density_difference_two_way: float = Field(
+        default=0.0001,
+        gt=0,
+        description='A number in kg/m3 indicating the minimum density difference above '
+        'which two-way flow may occur due to stack effect; required to run an '
+        'AirflowNetwork simulation. The default of 0.0001 is a typical default'
+        'value used for AirflowNetwork openings.'
+    )
+
+
+class AFNCrack(NoExtraBaseModel):
+    """Properties for airflow through a crack."""
+
+    type: constr(regex='^VentilationCrack$') = 'VentilationCrack'
+
+    air_mass_flow_coefficient_reference: float = Field(
+        ...,
+        gt=0,
+        description='A number in kg/s-m at 1 Pa per meter of crack length at the '
+        'conditions defined in the ReferenceCrack condition; required to run an '
+        'AirflowNetwork simulation. The DesignBuilder Cracks template defines the flow '
+        'coefficient for a tight, low-leakage wall to be 0.00001 and 0.001 for external '
+        'and internal constructions, respectively. Flow coefficients for a very poor, '
+        'high-leakage wall are defined to be 0.0004 and 0.019 for external and internal '
+        'constructions, respectively.'
+    )
+
+    air_mass_flow_exponent: float = Field(
+        default=0.65,
+        ge=0.5,
+        le=1,
+        description='An optional dimensionless number between 0.5 and 1 used to '
+        'calculate the crack mass flow rate; required to run an '
+        'AirflowNetwork simulation. This value represents the leak geometry impact '
+        'on airflow, with 0.5 generally corresponding to turbulent orifice flow and 1 '
+        'generally corresponding to laminar flow. The default of 0.65 is '
+        'representative of many cases of wall and window leakage, used when the '
+        'exponent cannot be measured.'
+    )
+
+    crack_factor: float = Field(
+        default=1,
+        gt=0,
+        le=1,
+        description='A number indicating multiplier for air mass flow through a crack.'
+    )
+
+
+class VentilationControlType(str, Enum):
+    single_zone = 'SingleZone'
+    multi_zone_with_distribution = 'MultiZoneWithDistribution'
+    multi_zone_without_distribution = 'MultiZoneWithoutDistribution'
+    multi_zone_with_distribution_only_during_fan_operation = \
+        'MultiZoneWithDistributionOnlyDuringFanOperation'
```

Comment:
Is this last one something that we are able to support right now? If not, maybe we should take it out until we add other objects needed for it to have a meaningful impact on the simulation.

Reply:
Yes it does depend on a detailed HVAC system, I can take it out.

Reply:
Sounds good.